app/views/pages/_editor_guide_text.html.erb

```diff
@@ -283,6 +283,36 @@
         <p>Add default-tab parameter to your CodePen embed tag. Defaults to <i>result</i>.</p>
         <pre>{% codepen https://codepen.io/twhite96/pen/XKqrJX default-tab=js,result %}</pre>
       </dd>
+
+      <dt><strong>preview</strong></dt>
```

Comment:
These "sub-headings" are not bold by default in the Liquid Tags docs; they are highlighted instead. Please change.

![Screen Shot 2022-01-18 at 11 45 40 AM](https://user-images.githubusercontent.com/32520970/149982305-54c8e8d2-e180-4aea-aeac-14925ed22664.png)

Reply:
Ah, I didn't notice that difference. `app/views/pages/_editor_guide_text.html.erb` is using `<strong>` on all of them,  `app/views/pages/_editor_liquid_help.html.erb` (in the screenshot) is using `<code>` though instead so my strong's are wrong there.

I'll update it in `app/views/pages/_editor_liquid_help.html.erb` so both files are consistent within themselves.

Happy to change it however you'd like, but I suspect this is what you'd want to see until these files are DRY'd up - lmk!